src/act_obj.c

```diff
@@ -16,23 +16,21 @@
 #include <sys/types.h>
 
 /* command procedures needed */
-DECLARE_DO_FUN(do_split);
-DECLARE_DO_FUN(do_yell);
-DECLARE_DO_FUN(do_say);
+void do_split(CHAR_DATA *ch, char *arg);
+void do_yell(CHAR_DATA *ch, char *arg);
+void do_say(CHAR_DATA *ch, char *arg);
 
 extern char *target_name; /* Included from magic.c */
 
 /*
  * Local functions.
  */
-#define CD CHAR_DATA
```

Comment:
Ditched one or two of these "short term" defines too, I'm not a fan.... There are more of these to deal with in merc.h later too.

Reply:
Totally. And I am pretty sure IDEs can do some of these!